scripts/dqlite/cmd/main.go

```diff
@@ -183,6 +200,42 @@ func main() {
 	<-ctx.Done()
 }
 
+func dumpDB(ctx context.Context, dial client.DialFunc, db *sql.DB, address, dir string) error {
+	cli, err := client.New(ctx, address, client.WithDialFunc(dial))
+	if err != nil {
+		return fmt.Errorf("client.New failed %w", err)
+	}
+
+	if dir == "" {
```

Comment:
I'm a little confused that in the outer function this was called "path" (which would imply foo/bar/baz.db) while inside this function it is just called "dir", which would imply "foo/bar".
I would at least be consistent and call the value "dir" in the outer function.